src/views/bugs.graphql

```diff
@@ -1,13 +1,17 @@
-query Bugs($goodFirst: BugSearch!, $mentored: BugSearch!, $paging: Paging) {
-  goodFirst: bugs(search: $goodFirst, paging: $paging) {
+query Bugs($goodFirst: BugSearch!, $mentored: BugSearch!, $pagingGood: Paging!, $pagingMen: Paging!) {
```

Comment:
Can we rename `pagingGood` to `goodFirstPaging` possibly and `pagingMen` to `mentoredPaging`? I think of the new name after the `$goodFirst` & `mentored` variables.